esx_service/vmdk_ops_test.py

```diff
@@ -193,6 +194,27 @@ def testAttachAsOpts(self):
         err = vmdk_ops.removeVMDK(self.name)
         self.assertEqual(err, None, err)
 
+    def test_VerifyVolumeSize(self):
+        """ test to verify the size of vmdk is same
+        as the vmdk's capacity reported by vmkfstools """
+        sizes = ['21gb', '20tb', '200mb']
+        opts = {}
+        for s in sizes:
+            err = vmdk_ops.createVMDK(vm_name=self.vm_name,
+                                      vmdk_path=self.name,
+                                      vol_name=self.volName,
+                                      opts={volume_kv.SIZE:s})
+            self.assertEqual(err, None, err)
+            self.assertEqual(os.path.isfile(self.name), True,
+                             "VMDK {0} is missing after create.".format(self.name))
+            response = vmdkops_admin.get_vmdk_size_info(path=self.name)
+            head = s.rstrip('bgtmk')
+            tail = s[len(head):]
+            self.assertEqual(response["capacity"], "{0:.2f}".format(float(head))+ tail.upper(),\
```

Comment:
we can simply compare the outcome of response["capacity"] with the expected in this case 's' .. there is no need for extra steps of head and tail ..